docker/accelerate-cpu/Dockerfile

```diff
@@ -27,8 +27,6 @@ RUN python3 -m pip install --no-cache-dir \
 # Stage 2
 FROM python:3.8-slim AS build-image
 COPY --from=compile-image /opt/venv /opt/venv
-RUN useradd -ms /bin/bash user
-USER user
```

Comment:
Generally for security this is bad practice to not create a user in the docker image. Is there another solution we can try instead?

Reply:
Thank you for your feedback, and I fully agree with your point. Not setting up a non-root user was indeed a critical issue. Instead, I’ve configured the container to grant limited `sudo` permissions for `apt-get` and `pip` commands to the user, which allows necessary installations while maintaining security.

I’ve verified that the package is accessible within the container as expected:

```bash
user@f6ece721f352:/workspaces/accelerate$ pip show accelerate
Name: accelerate
Version: 1.1.0.dev0
Summary: Accelerate
Home-page: https://github.com/huggingface/accelerate
Author: The HuggingFace team
Author-email: zach.mueller@huggingface.co
License: Apache
Location: /opt/venv/lib/python3.9/site-packages
Requires: huggingface-hub, numpy, packaging, psutil, pyyaml, safetensors, torch
Required-by:
```

Thanks for guiding me towards a more secure solution!